### ISSUE TITLE
Is this a bug?

### ISSUE THREAD

**Issue by shwuhk** [generEOS/poorman.token#5]
https://github.com/generEOS/poorman.token/blob/f55a1dbe8763f0735c4cb6f25a8160788f87a78a/poorman.token.cpp#L186

In this line, it should be pay_ram == false instead of pay_ram == true?
From my best understanding, if pay_ram is set to true, even the user does not have a record yet, the `from account` would still want to pay the fee to place the record instead of assert.

**Comment by Khazius** [comment]
I was experimenting with some changes recently, though i think I will branch it and revert the commit to avoid confusion.

The concept here was to guarantee an airdrop is always free by specifying that the issue or transfer must have zero cost. traditional functions still work like normal.

**Comment by shwuhk** [comment]
Yes, I know what you mean. But is the assert condition inverted? Isn't it be pay_ram == false instead of pay_ram == true?

**Comment by Khazius** [comment]
The assert is correct. In that part of the if statement we will create a new row. We only want to create the row if ```pay_ram == true```. We assert that``` pay_ram==true``` to proceed. If ```pay_ram==false``` the assert fails and we don't create a new row.
